schema/release-schema.json

```
@@ -1224,28 +1224,28 @@
         },
         "description": {
           "title": "Budget Source",
-          "description": "A short free-text description of the budget allocation for this contracting process. This may be used to provide human-readable information on the budget category allocated to this contracting process, and/or, information about the nature and source of the allocation (e.g. conditional, confirmed; any official authorizations given to the budget allocation).",
+          "description": "A short free-text description of the budget allocation which will be used in a future contracting process. This may be used to provide human-readable information on the budget category allocated to the future contracts, and/or, information about the nature and source of the allocation (e.g. conditional, confirmed; any official authorizations given to the budget allocation).",
```

To match wording of `amount`.

```suggestion
          "description": "A short free-text description of the budget to be used in a future contracting process. This may be used to provide human-readable information on: the budget category for the future contracts and/or the nature and source of the budget allocation (e.g. conditional, confirmed, or any official authorizations given to the budget allocation).",
```